x/concentrated-liquidity/model/codec.go

```diff
@@ -13,6 +15,12 @@ func RegisterCodec(cdc *codec.LegacyAmino) {
 }
 
 func RegisterInterfaces(registry cdctypes.InterfaceRegistry) {
+	registry.RegisterInterface(
+		"osmosis.concentratedliquidity.v1beta1.PoolI",
+		(*swaproutertypes.PoolI)(nil),
+		&Pool{},
+	)
```

Comment:
From previous discussions - I'm still unsure if this is needed

Reply:
Removed, was there for testing purposes. Still unsure how to get this work in localosmosis without it, so will just continue to keep it there for local testing for now